pom.xml

```diff
@@ -1101,5 +1101,61 @@
         </plugins>
       </build>
     </profile>
+    <!-- Profiles preconfiguring different Jenkins versions. -->
+    <!-- LTS lines are used. -->
+    <profile>
+      <id>jenkins-642</id>
+      <properties>
+        <jenkins.version>1.642.1</jenkins.version>
+        <hpi-plugin.version>1.115</hpi-plugin.version>
+        <stapler-plugin.version>1.17</stapler-plugin.version>
+        <java.level>7</java.level>
+      </properties>
+    </profile>
+    <profile>
+      <id>jenkins-625</id>
+      <properties>
+        <jenkins.version>1.625.3</jenkins.version>
```

Comment:
The idea of the profiles is for them to be available for testing on the command line, not to configure the default build, e.g.:

```
mvn -Pjenkins-625 clean verify
```

This can be used to test building against different LTS versions for those interested in doing so. If we activate some by default and then the user activates another the user has to start thinking about which properties values take preference (as there are more than one profile contributing values), so in the end is better not to depend on them for the default use case.